packages/client/src/modules/Storage.ts

```diff
@@ -19,7 +19,7 @@ export class StorageModule {
     return safeResults.value;
   }
 
-  async set(key: string, value: string, storeId?: string) {
+  async set(key: string, value: string, storeId?: string): Promise<void> {
```

Comment:
This should not be needed as we infer the type from the return value unless you prefer to add it explicitly
```suggestion
  async set(key: string, value: string, storeId?: string) {
```